src/Tizen.NUI/src/public/BaseComponents/VectorGraphcis/Shape.cs

```diff
@@ -269,7 +269,7 @@ public bool AddRect(float x, float y, float width, float height, float roundedCo
         /// <param name="radiusX">X axis radius of the circle.</param>
```

Comment:
I would say that if has 2 radiuses, then it's not a Circle but rather an Ellipse.